src/groupByKey.ts

```diff
@@ -2,6 +2,15 @@ type GroupsMap<T> = {
   [key: string]: T[];
 };
 
-export function groupByKey(items, key) {
-  // write code here;
+export function groupByKey<T>(items: T[], key: keyof T): GroupsMap<T> {
+  const result: GroupsMap<T> = {};
+
+  items.forEach((item) => {
+    const keyValue = item[key] ? (item[key] as string) : '';
```

Comment:
```suggestion
    const keyValue = item[key] || '';
```

Reply:
![Uploading Screenshot 2023-11-14 at 15.08.49.png…]()

Reply:
we have a problem here 
<img width="1027" alt="Screenshot 2023-11-14 at 15 08 56" src="https://github.com/mate-academy/ts_group_by_key/assets/112933414/c1e37dbd-c5b2-43b7-ad13-41c5e0388738">
<img width="1098" alt="Screenshot 2023-11-14 at 15 08 49" src="https://github.com/mate-academy/ts_group_by_key/assets/112933414/e4921e2e-db88-4c46-979d-c30dc3da3e92">

Reply:
okay, let's try this one 
`const keyValue = item[key] as string || ''`;